music21/musicxml/helpers.py

```diff
@@ -179,6 +182,42 @@ def splitSuffix(measureNumber):
         sortedSuffixes = sorted([m1Suffix, m2Suffix])
         return m1Suffix is sortedSuffixes[0]
 
+def childrenEqual(e1: Element, e2: Element, children: List[str]) -> bool:
```

Comment:
this is great -- but I feel more that we should be removing in music21 before generating musicxml -- when musicxml is manipulated after generation it becomes much harder to debug -- "There were two <key> tags...oh, now there's one"